src/js/stores/__tests__/DCOSStore-test.js

```diff
@@ -336,7 +336,7 @@ describe("DCOSStore", function() {
       ]);
 
       expect(DCOSStore.serviceTree.getItems()[0].getStatus()).toEqual(
-        "Recovering"
```

Comment:
Some services that until now had "Recovering" status now have a "Delayed" status.